Services/UI/OJS.Services.Ui.Business/Validation/Implementations/ContestValidationService.cs

```diff
@@ -36,18 +31,14 @@ public ValidationResult GetValidationResult((Contest?, string, bool, bool) item)
                 userId,
                 isUserAdmin,
                 isUserLecturerInContest,
-                allowToAdminAlways: true))
+                true))
```

Comment:
Leave it like before - 'allowToAdminAlways: true' to easily understand what this true stands for.